server/api/restaurants.js

```diff
@@ -36,22 +36,17 @@ router.get('/:id', (req, res, next) => {
 })
 
 //by Geolocation
-router.get('/location/:lat/:long', (req, res, next) => {
```

Comment:
this might be subjective but i think in this case, query string would be better. https://stackabuse.com/get-query-strings-and-parameters-in-express-js/